routes/v1.js

```diff
@@ -3,8 +3,11 @@ import UserController from '../controllers/user';
 import ProfileController from '../controllers/profile';
 import Auth from '../middleware/auth';
 import isUserVerified from '../middleware/verifyUser';
+// import verifyUser from '../middleware/verifyUser';
```

Comment:
@chingsley if this code is not being used, it should be removed

Reply:
Thanks. I have removed it.